bakery/checker.go

```diff
@@ -35,57 +27,82 @@ type Op struct {
 	Action string
 }
 
+// NoOp holds the empty operation, signifying no authorized
+// operation. This is always considered to be authorized.
+var NoOp = Op{}
```

Comment:
i think an operation that is always considered un-authorized also makes sense...

Reply:
a note like "see OpsAuthorizer for its use" would be in order...

Reply:
done